packages/react/src/primitives/Expander/Expander.tsx

```diff
@@ -9,7 +9,7 @@ import { ExpanderProps } from '../types/expander';
 import { Primitive } from '../types/view';
 import { splitPrimitiveProps } from '../shared/styleUtils';
 
-const ExpanderPrimitive: Primitive<ExpanderProps, typeof Root> = (
+const ExpanderPrimitive: Primitive<ExpanderProps, 'div'> = (
```

Comment:
This makes the Expander html element type more explicit. Before TS was having trouble finding the type, which was causing a TS strict error on the ref param.